### PR TITLE
Alkymi (Non-Microsoft Publisher)

### PR DESCRIPTION
**Pull request by tori-alkymi**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thanks, @tori-alkymi for your first Independent Publisher connector submission! We will be reviewing your files shortly.

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:6]: @tori-alkymi Please provide a detailed description that everyone can understand and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:40]: @tori-alkymi Please provide a valid description value for the response code, and you can all the possible response codes of the operation.

Applies the same comment for all the similar properties in the swagger.

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:171]: @tori-alkymi Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:460]: @tori-alkymi Empty property values can be removed unless you have a specific reason to keep them. Applies the same comment for all the similar empty properties in the swagger.

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:913]: @tori-alkymi All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:1]: @tori-alkymi If it is an independent publisher connector, the connector artifacts should be placed under the "independent-publisher connectors" folder.

File: certified-connectors/Alkymi/readme.md
Comment (sriyen-msft) [inline certified-connectors/Alkymi/readme.md:6]: @tori-alkymi Please find a template for readme.md file [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates) and add the required sections.

File: certified-connectors/Alkymi/readme.md
Comment (sriyen-msft) [inline certified-connectors/Alkymi/readme.md:13]: @tori-alkymi Please fix the formatting of the readme.md file by referring to the template.

File: certified-connectors/Alkymi/apiProperties.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiProperties.json:52]: @tori-alkymi If it is an independent publisher connector, the publisher value should be your name.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by tori-alkymi** [comment]
Hi, I accidentally put the wrong tag on the title initially - this is actually a Non-Microsoft publisher connector. I've made the requested changes and have one more question. When I run `paconn validate`, I get the following warning that I'm not sure how to resolve:

`Swagger certification succeeded with warnings. It is not required but recommended to fix following warnings: Warning : paths/~1api~1v2~1webhooks~1/post/parameters/0/schema/properties/target : The notification URL property must be required all the way through the schema chain.`

Thanks so much for your help and let me know what you think!

**Comment by tori-alkymi** [comment]
@sriyen-msft Can you please re-review this?

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [reply certified-connectors/Alkymi/apiDefinition.swagger.json:6]: @tori-alkymi Can you please let us know why this comment was resolved?

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:1234]: @tori-alkymi Please provide a valid description value for the property.

Applies the same comment for all the similar properties in the swagger.

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:1219]: @tori-alkymi There are many incorrect description values in the swagger. All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [reply certified-connectors/Alkymi/apiDefinition.swagger.json:1]: @tori-alkymi Can you please let us know why this comment was resolved?

File: certified-connectors/Alkymi/readme.md
Comment (sriyen-msft) [reply certified-connectors/Alkymi/readme.md:6]: @tori-alkymi Can you please let us know why this comment was resolved?

File: certified-connectors/Alkymi/apiProperties.json
Comment (sriyen-msft) [reply certified-connectors/Alkymi/apiProperties.json:52]: @tori-alkymi Can you please let us know why this comment was resolved?

File: certified-connectors/Alkymi/readme.md
Comment (sriyen-msft) [reply certified-connectors/Alkymi/readme.md:22]: @tori-alkymi Can you please let us know why this comment was resolved?

**Comment by sriyen-msft** [comment]
@tori-alkymi Apologies for the delay in response, We have a high number of submissions and had to prioritize the connectors which were ready for production deployment. We are reviewing submissions one by one sorted by created date time.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (tori-alkymi) [reply certified-connectors/Alkymi/apiDefinition.swagger.json:1]: Because this is a submission for a certified connector.

**Comment by tori-alkymi** [comment]
@sriyen-msft 

It looks like I missed updating several of the descriptions. Those should be updated now, but please let me know if you see any remaining issues. Also, as mentioned on a comment above, this connector is being submitted as a Non-Microsoft Publisher connection, so I've been working off the readme template here:

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md

Please let me know if I should be referring to a different template. On a related note, that is why the connector is placed in the certified-connectors folder. Thanks again for all your help!

**Comment by tori-alkymi** [comment]
Also, I'm getting some feedback from our product team, so I'm going to wait on requesting re-review until I get a go-ahead from them.

**Comment by sriyen-msft** [comment]
> @sriyen-msft
> 
> It looks like I missed updating several of the descriptions. Those should be updated now, but please let me know if you see any remaining issues. Also, as mentioned on a comment above, this connector is being submitted as a Non-Microsoft Publisher connection, so I've been working off the readme template here:
> 
> https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md
> 
> Please let me know if I should be referring to a different template. On a related note, that is why the connector is placed in the certified-connectors folder. Thanks again for all your help!

I am not seeing major issues with your PR right now apart from the comments I have posted. Feel free to reach out to us if you need any help on your connector certification. Thanks.

File: certified-connectors/Alkymi/readme.md
Comment (sriyen-msft) [reply certified-connectors/Alkymi/readme.md:6]: @tori-alkymi You have missed to add author section

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [reply certified-connectors/Alkymi/apiDefinition.swagger.json:6]: @tori-alkymi Please provide a detailed description that everyone can understand and it should describe concisely the main purpose and value offered by your connector.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@tori-alkymi I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by tori-alkymi** [comment]
@sriyen-msft I think everything should be in good shape now. Let me know if you see any remaining issues and thanks again!

File: certified-connectors/Alkymi/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Alkymi/apiDefinition.swagger.json:6]: @tori-alkymi Please remove html tags of newline (\n) from description values, they are just placed randomly in the description.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by tori-alkymi** [comment]
@sriyen-msft

**Comment by tori-alkymi** [comment]
@sriyen-msft I heard that you were out for a family issue and I hope everything is okay! When you get back, could you please take another look at this PR?

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**
